docs/source/R-api.rst

```diff
@@ -19,7 +19,10 @@ Install MLflow
 
 Installs auxiliary dependencies of MLflow (e.g. the MLflow CLI). As a
 one-time setup step, you must run install_mlflow() to install these
-dependencies before calling other MLflow APIs.
+dependencies before calling other MLflow APIs. Alternatively, you can set MLFLOW_PYTHON_BIN and
+MLFLOW_BIN environment variables instead. These variables allow you to use custom mlflow
```

Comment:
Do we document which endpoints the r_serve exposes?

Reply:
This PR does not deal with r serve.